README.md

```diff
@@ -126,11 +126,16 @@ Sets the duration of vaccine efficacy in the model
 outputs.
 
 ```js
-function runModel(parameters, atol=1e-3, rtol=1e-3)
+function runModel(parameters, control)
 ```
 
-The atol and rtol parameters are for absolute and relative tolerances for the
-ODE solver.
+The `control` paramer is a dict that may take elements:
+
+* `atol`: absolute tolerance (e.g., 1e-5)
+* `rtol`: relative tolerance (e.g., 1e-5; generally set this to the same as `atol`)
+* `stepSizeMin`: smallest allowable step size
+* `stepSizeMinAllow`: boolean, if `true` then we continue after hitting `stepSizeMin` and do not reduce the step size further
```

Comment:
Just for understanding this then throws `throw integrationError("step too small", t);` if set to false and the min is hit. IF true an the min is hit it will carry on at that step size.

Reply:
That's right; see here for details https://github.com/mrc-ide/dopri-js/pull/10, or equivalently in R https://github.com/mrc-ide/dde/pull/27